app/addons/cors/components/OriginTable.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import OriginRow from "./OriginRow";
+
+export default class OriginTable extends Component {
+
+  constructor (props) {
+    super(props);
+  }
+
+  createRows () {
+    return _.map(this.props.origins, function (origin, i) {
```

Comment:
If origins is an array can you rather use array.map. If its an object, you can leave it.